assets/base.css

```diff
@@ -2351,7 +2355,7 @@ product-info .loading-overlay:not(.hidden) ~ *,
   .header--top-center.drawer-menu {
     grid-template-areas:
       'left-icons heading icons';
-    grid-template-columns: 1fr 2fr 1fr;
+    grid-template-columns: 1fr auto 1fr;
```

Comment:
Same here, since we're on desktop and large screens, the logo can never be greater than 300px. So it makes that column take only the amount of space that's needed.